examples/Sentinel_2_example_notebook/utils.py

```diff
@@ -0,0 +1,184 @@
+import numpy as np
+import scipy
+import rasterio
+
+# [1] Massimetti, Francesco, et al. ""Volcanic hot-spot detection using SENTINEL-2:
```

Comment:
The code is a simplified version of the one described by that paper. It belongs to our repo, which is at the moment private and be released soon. I will acknowledge the project and the repo as soon as we release it.

Reply:
But what license will it be released under and what license is it currently under? If it has no license it is by default proprietary and we cannot just publish it.

Reply:
The code belongs to our repo that reimplements the code described by "Massimetti et al.". The repo will be released soon with GPL license. I have put a comment to clarify this.

Reply:
Please add the license here.

Reply:
Just put this below the description
```
This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.

    This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.

    You should have received a copy of the GNU General Public License along with this program. If not, see <https://www.gnu.org/licenses/>. 
```

Reply:
??? Still not added